Modelplace_Cloud_API/argparser.py

```diff
@@ -0,0 +1,46 @@
+import argparse
+
+
+def parse_args() -> argparse.Namespace:
+    parser = argparse.ArgumentParser(
+        description='Example of a Python application using the Modelplace.AI Cloud API'
+    )
+    parser.add_argument(
+        "-e",
```

Comment:
-u?

Reply:
I think it is better to use "-e" because the login requires an email, not a username.